tests/unit/BenCorrection.spec.ts

```diff
@@ -0,0 +1,397 @@
+import Vue from 'vue'
```

Comment:
This is the original Correction.spec.ts file, except it imports/mounts BenCorrection.vue instead of Correction.vue.